AngleSharp.Io.Tests/AngleSharp.Io.Tests.csproj

```diff
@@ -8,7 +8,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>AngleSharp.Io.Tests</RootNamespace>
     <AssemblyName>AngleSharp.Io.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
I think its alright, but I made 4.6 on purpose. I figured out 4.5 would be a good choice for the desktop - may target more projects than 4.6. The test project is only used during development, where 4.6 should be available. Of course it does not matter here as long as all features are available (4.5 is sufficient) and the version is >= the references library.